backend/geneaprove/views/to_json.py

```diff
@@ -169,7 +169,7 @@ def get(self, request, *args, **kwargs):
         """
         params = JSONViewParams()
         params.update(request.GET)
-        logger.debug('%s.%s.get(%s)', self.__module__, self.__class__.__name__, params)
+        # logger.debug(f'{self.__module__}.{self.__class__.__name__}.get({params})')
```

Comment:
I find those convenient for debugging, any reason to remove them ?

Reply:
Neither this nor the one in the next message was removed. I had commented them during my own investigations and forgot to bring them back out of that status for my commits.

They're both now f-strings and uncommented. Corrected.